tests/conftest.py

```diff
@@ -2867,21 +2868,33 @@ def ns_resource_factory(request, mcg_obj, cld_mgr, cloud_uls_factory):
 
     """
     created_ns_resources = []
-    created_ns_connections = []
+    created_connections = {}
 
-    def _create_ns_resources():
+    def _create_ns_resources(platform=constants.AWS_PLATFORM):
```

Comment:
Why does it make sense to select AWS as predefined platform for the tests?

Reply:
Because I don't want to break any existing functionality and tests. I am not only one that uses these functions and I am not planning to update all tests to have explicitly selected platform in this PR.

Reply:
Makes sense.